app/services/embed.rb

```diff
@@ -63,6 +65,16 @@ def build_type
       end
     end
 
+    def build_media_type
+      if !viewer_enabled?
+        "Download"
+      elsif file_set&.video?
+        "Video"
+      else
+        "Image"
```

Comment:
I think this will result in an audio resource having `media_type` "Image". It won't be a problem technically but it is a bit incongruous, what do you think?

Reply:
Yeah I just didn't add it because I didn't have a use case for it, but I can.